elide-datastore/elide-datastore-search/README.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+Alpha version of full text search for Elide.  
+
+## Requirements
+
+This store leverages [Hibernate Search](https://hibernate.org/search/) which requires Hibernate 5.4+.
+
+## Usage
+
+`SearchDataStore` wraps another fully featured store and supports full text search on fields that are indexed using Hibernate Search.
+If the query cannot be answered by the `SearchDataStore`, it delegates the query to the underlying (wrapped) data store.
+
+### Annotate Your Entity 
+
+Use Hibernate Search annotations to describe how your entities are indexed and stored in Lucene or Elasticsearch.
+
+```java
+@Entity
+@Include
+@Indexed
+@Data
+@AnalyzerDef(name = "case_sensitive",
+        tokenizer = @TokenizerDef(factory = StandardTokenizerFactory.class)
+)
+@AnalyzerDef(name = "case_insensitive",
```

Comment:
Can we mention that these annotations can be at the `package-info.java`  to remove the boilerplate?

Reply:
👍